migration/i7.1z/oracle/202009171400_IDEMPIERE-4452.sql

```diff
@@ -0,0 +1,51 @@
+SET SQLBLANKLINES ON
+SET DEFINE OFF
+
+-- IDEMPIERE-4452
+-- Sep 17, 2020, 8:08:18 AM UTC
+INSERT INTO AD_Element (AD_Element_ID,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,ColumnName,Name,Description,Help,PrintName,EntityType,AD_Element_UU) VALUES (203439,0,0,'Y',TO_DATE('2020-09-17 08:08:17','YYYY-MM-DD HH24:MI:SS'),100,TO_DATE('2020-09-17 08:08:17','YYYY-MM-DD HH24:MI:SS'),100,'IsIgnoreIsCurrentVendor','Ignore Current Vendor Flag','take all PO prices into account','will use PO price even if it is not marked as current vendor','Ignore Current Vendor Flag','D','2db796b8-1da0-4c9f-b49f-7c63659b5de3')
+;
+
+-- Sep 17, 2020, 8:15:42 AM UTC
+INSERT INTO AD_Column (AD_Column_ID,Version,Name,Description,Help,AD_Table_ID,ColumnName,DefaultValue,FieldLength,IsKey,IsParent,IsMandatory,IsTranslated,IsIdentifier,SeqNo,IsEncrypted,AD_Reference_ID,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,AD_Element_ID,IsUpdateable,IsSelectionColumn,EntityType,IsSyncDatabase,IsAlwaysUpdateable,IsAutocomplete,IsAllowLogging,AD_Column_UU,IsAllowCopy,SeqNoSelection,IsToolbarButton,IsSecure,FKConstraintType,IsHtml) VALUES (214315,0,'Ignore Current Vendor Flag','take all PO prices into account','will use PO price even if it is not marked as current vendor',477,'IsIgnoreIsCurrentVendor','N',1,'N','N','Y','N','N',0,'N',20,0,0,'Y',TO_DATE('2020-09-17 08:15:41','YYYY-MM-DD HH24:MI:SS'),100,TO_DATE('2020-09-17 08:15:41','YYYY-MM-DD HH24:MI:SS'),100,203439,'Y','N','D','N','N','N','Y','713ddf8e-f924-46a7-a9c2-11003c399dad','Y',0,'N','N','N','N')
+;
+
+-- Sep 17, 2020, 8:16:18 AM UTC
+INSERT INTO AD_Column (AD_Column_ID,Version,Name,Description,Help,AD_Table_ID,ColumnName,FieldLength,IsKey,IsParent,IsMandatory,IsTranslated,IsIdentifier,SeqNo,IsEncrypted,AD_Reference_ID,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,AD_Element_ID,IsUpdateable,IsSelectionColumn,EntityType,IsSyncDatabase,IsAlwaysUpdateable,IsAutocomplete,IsAllowLogging,AD_Column_UU,IsAllowCopy,SeqNoSelection,IsToolbarButton,IsSecure,IsHtml) VALUES (214316,0,'Partner Category','Product Category of the Business Partner','The Business Partner Category identifies the category used by the Business Partner for this product.',477,'VendorCategory',30,'N','N','N','N','N',0,'N',10,0,0,'Y',TO_DATE('2020-09-17 08:16:18','YYYY-MM-DD HH24:MI:SS'),100,TO_DATE('2020-09-17 08:16:18','YYYY-MM-DD HH24:MI:SS'),100,622,'Y','N','D','N','N','N','Y','d3bd3b17-d48d-43d7-9d79-f58aad219bcc','Y',0,'N','N','N')
+;
+
+-- Sep 17, 2020, 8:19:51 AM UTC
+ALTER TABLE M_DiscountSchemaLine ADD IsIgnoreIsCurrentVendor CHAR(1) DEFAULT 'N' CHECK (IsIgnoreIsCurrentVendor IN ('Y','N')) NOT NULL
+;
+
+-- Sep 17, 2020, 8:20:37 AM UTC
+ALTER TABLE M_DiscountSchemaLine ADD VendorCategory VARCHAR2(30) DEFAULT NULL 
```

Comment:
we have change to use 30 char (i.e Varchar2(30 Char)) for Oracle

Reply:
Ok.